dre.json

```diff
@@ -0,0 +1,28 @@
+{
+	"response": [
+		{
+			"maintainer": "Dan (rosand25)",
+			"oem": "OnePlus",
+			"device": "OnePlus Nord N200",
+			"filename": "crDroidAndroid-13.0-20230920-dre-v9.9.zip",
+			"download": "https://sourceforge.net/projects/crdroid/files/dre/9.x/crDroidAndroid-13.0-20230920-dre-v9.9.zip/download",
+			"timestamp": 1695184539,
+			"md5": "85cc9befaeaf7bff74d334823c34c85f",
+			"sha256": "e7ba5a4a31dcbc9a8a3b8c13cf54745cebdfffc0260a8ee1d0bbafc0c3729988",
+			"size": 1172670238,
+			"version": "9.9",
+			"buildtype": "Weekly",
+			"forum": "https://forum.xda-developers.com/t/rom-unofficial-dre-13-crdroid-v9-9-for-oneplus-nord-n200-5g.4629577/",
+			"gapps": "https://github.com/MindTheGapps/13.0.0-arm64/releases",
+			"firmware": "https://sourceforge.net/projects/crdroid/files/dre/9.x/firmware",
+			"modem": "",
+			"bootloader": "",
+			"recovery": "https://sourceforge.net/projects/crdroid/files/dre/9.x/recovery",
+			"paypal": "",
+			"telegram": "",
+			"dt": "https://github.com/rosand25/android_device_oneplus_dre",
```

Comment:
use crDroid repos not personal one
https://github.com/crdroidandroid/android_device_oneplus_dre